pkgs/development/haskell-modules/make-package-set.nix

```diff
@@ -281,46 +281,28 @@ in package-set { inherit pkgs stdenv callPackage; } self // {
       let
```

Comment:
I know this isn't directly related to this PR, but could you add a small comment about what other arguments you can pass to `shellFor`?

For example, I think most people will be interested in `buildInputs`, where you can add things like `cabal-install` and `ghcid`.

Reply:
Added a small example in the newest changes